nil/core-concepts/contract-co-location.mdx

```diff
@@ -10,11 +10,9 @@ In plain terms, co-location is a request made to change the value stored under t
 
 :::note[Explanation]
 
-When two accounts (Account 1 and Account 2) are co-located, communications between them can occur synchronously similarly to regular Ethereum transactions. 
+When two contracts (Contract 1 and Contract 2) are co-located, communications between them can occur synchronously without spawning messages. However, if Contract 1 and Contract 2 are located on different shards, communications between them will always be asynchronous. Async calls between contracts spawn additional messages, and their execution times may vary.
```

Comment:
Let's just say that we have a way to deploy contracts to specific shards. As of now, we don't have a mechanism of mapping contracts to shards other than initial deployment (meaning that we can't move contracts between shards). 

We may have "colocation" in the future. But for now, mentioning colocation will just confuse users and they will ask us how they can utilize it.

Reply:
The synchronous/asynchronous communication is this paragraph is described correctly. So just need to reword the "colocation"